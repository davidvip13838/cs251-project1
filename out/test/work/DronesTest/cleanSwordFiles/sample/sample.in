2 4 5
0
4
0
1
1
3
