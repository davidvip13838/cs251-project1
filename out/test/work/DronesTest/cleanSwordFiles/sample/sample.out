0 0
4 3
9 8
14 11
